### ISSUE TITLE
If graph and final graph share an id, the id must be stored only once

### ISSUE THREAD

**Issue by rekola** [Sometrik/graphlib#44]
You could have a graph holder class that contains everything that's common to both graphs:
- id
- source_id
- object counters
- name
- keywords
- server_search_id
- is_loaded
- line_width
- default_symbol_id
- show_nodes, show_edges, show_faces, show_labels
- node_color, edge_color, face_color

Or things that are not needed for both:
- statistics
- filter
- alpha
- child_alpha

After this, consider if the createSimilar() method could be removed.
